node/src/service.rs

```diff
@@ -43,20 +50,45 @@ native_executor_instance!(
 ///
 /// Use this macro if you don't actually need the full service, but just the builder in order to
 /// be able to perform chain operations.
-pub fn new_partial(
+pub fn new_partial<RuntimeApi, Executor, BIQ>(
     config: &Configuration,
+    build_import_queue: BIQ,
 ) -> Result<
     PartialComponents<
         TFullClient<Block, RuntimeApi, Executor>,
         TFullBackend<Block>,
         (),
-        sp_consensus::import_queue::BasicQueue<Block, PrefixedMemoryDB<BlakeTwo256>>,
+        sc_consensus::DefaultImportQueue<Block, TFullClient<Block, RuntimeApi, Executor>>,
         sc_transaction_pool::FullPool<Block, TFullClient<Block, RuntimeApi, Executor>>,
-		(Option<sc_telemetry::Telemetry>, Option<sc_telemetry::TelemetryWorkerHandle>),
+        (Option<Telemetry>, Option<TelemetryWorkerHandle>),
     >,
     sc_service::Error,
-> {
-    let inherent_data_providers = sp_inherents::InherentDataProviders::new();
+>
+    where
+        RuntimeApi: ConstructRuntimeApi<Block, TFullClient<Block, RuntimeApi, Executor>>
+        + Send
```

Comment:
suggest indent this like https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/service.rs#L65

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231